lua_modules/http/http-example.lua

```diff
@@ -21,17 +23,16 @@ require("httpserver").createServer(80, function(req, res)
     -- end
   end
   -- setup handler of body, if any
-  req.ondata = function(self, chunk)
+  req.ondata = function(self, chunk, last)
     print("+B", chunk and #chunk, node.heap())
-    if not chunk then
+    if last then
```

Comment:
As said elsewhere, I'd prefer the original API.

Reply:
Me too, It would be more consistent with `send` as you said (elsewhere?)